app/controllers/settings/users/index.js

```diff
@@ -83,16 +76,6 @@ export default class UsersSettingsController extends Controller {
     this.filter = this.searchTextBuffer;
   }
 
-  @task
-  *loadSelectedOrganizations() {
-    this.selectedOrganizations = yield Promise.all(this.organizations.map((id) => this.store.findRecord('user-organization', id)));
-  }
-
-  @task
-  *loadSelectedRoles() {
-    this.selectedRoles = yield Promise.all(this.roles.map((id) => this.store.findRecord('role', id)));
-  }
```

Comment:
Can we keep those tasks and just call them from the route's `setupController`? As such, we can still show a loading state in the template, without the entire page rendering being blocked (which is the case when data is being loaded in the `afterModel`-hook).